charts/redpanda/templates/statefulset.yaml

```diff
@@ -85,6 +85,9 @@ spec:
             runAsUser: 0
             runAsGroup: 0
           volumeMounts:
+{{- if get .Values.statefulset.initContainers.tuning "extraVolumeMounts" -}}
```

Comment:
initContainers on down are not required parameters in the schema. We should either require them or use dig for each of these:
```suggestion
{{- if dig  "initContainers" "tuning" "extraVolumeMounts" false .Values.statefulset -}}
```

Reply:
oh i see what you mean, alright.